guides/common/attributes.adoc

```diff
@@ -24,6 +24,9 @@ ifeval::["{build}" == "foreman"]
 :foreman-example-com: foreman.example.com
 :installer-scenario: foreman-installer --scenario katello
 :installer-scenario-smartproxy: foreman-installer --no-enable-foreman
+:install: yum install
+:clean: yum clean
+:remove: yum remove
```

Comment:
I'd probably name these `package-install`, those look too generic to me.

Reply:
Also clean can have different meaning for other package managers, I'd probably not bother in that particular case.

Reply:
Let's keep this and ask the community members working with Debian to let us know if an alternative command would work for them.